MSAL/src/MSALPublicClientApplication.m

```diff
@@ -1434,12 +1434,27 @@ - (void)signoutWithAccount:(nonnull MSALAccount *)account
         block(NO, localError, nil);
         return;
     }
+
+    if(signoutParameters.wipeCacheForAllAccounts)
```

Comment:
Code style: add a space after if, should be:

if (signoutParameters.wipeCacheForAllAccounts)

Reply:
Fixed